packages/dds/tree/src/shared-tree/revertible.ts

```diff
@@ -30,6 +34,11 @@ export interface Revertible {
 	 */
 	revert(dispose: boolean): void;
 
+	/**
+	 * Forks a new revertible from the current revertible.
+	 */
+	fork(view: RervertibleTarget): Revertible;
```

Comment:
What is this revertible target, exactly? I was expecting to see the existing `TreeView` or `SchematizingSimpleTreeView`... but I guess this is happening at a lower level than those exist, so maybe that's why?